test/LinalgExt/tiling.mlir

```diff
@@ -44,3 +43,51 @@ func @reverse_1d_tensor(%arg0 : tensor<?x?xf32>) -> tensor<?x?xf32> {
 //      CHECK:       scf.yield %[[INSERT_SLICE]]
 //      CHECK:     scf.yield %[[YIELD]]
 //      CHECK:   return %[[RESULT]]
+
+// -----
+
+func @gemm(%lhs : tensor<?x?xf32>, %rhs : tensor<?x?xf32>,
```

Comment:
nit: matmul is not gemm

Reply:
`linalg.matmul` is almost gemm (missing the scalar values :P )